__mocks__/gatsby.js

```diff
@@ -0,0 +1,27 @@
+// https://www.gatsbyjs.org/docs/unit-testing/
+const React = require("react")
+const gatsby = jest.requireActual("gatsby")
+module.exports = {
```

Comment:
@jasonLaster has done some legwork in #218 to set up real integration tests for the frontend. My preference is to use Jest for "pure" JS functions as you've done in the case of `makeSlug` and keep the React testing in Cypress.

I think we can drop the default Gatsby mocks.